yt_dlp/extractor/mux.py

```diff
@@ -0,0 +1,33 @@
+from .common import InfoExtractor
+
+
+class MuxStreamNewIE(InfoExtractor):
+    IE_NAME = 'stream.new'
```

Comment:
Since this all the same infrastructure, might as well name it `MuxIE`
```suggestion
class MuxIE(InfoExtractor):
```